app_server/app/adapters/socket/RoomsManager.js

```diff
@@ -0,0 +1,105 @@
+/**
+ * @module RoomsManager
+ * Manage commenting and sending sticker functionalities
+ */
+'use strict';
+
+var rfr = require('rfr');
+
+var Utility = rfr('app/util/Utility');
+var RequestInjector = rfr('app/adapters/socket/RequestInjector');
+var Room = rfr('app/adapters/socket/Room');
+var Client = rfr('app/adapters/socket/Client');
+
+var logger = Utility.createLogger(__filename);
+
+var DEFAULT_ROOM = 'lobby';
+
+function RoomsManager(server, io) {
+  this.io = io;
+
+  this.rooms = {};
+  this.rooms[DEFAULT_ROOM] = new Room(DEFAULT_ROOM,
+                                      Room.ROOM_TYPES.GENERAL);
+  this.users = {};
+
+  this.requestInjector = new RequestInjector(server);
+}
+
+var Class = RoomsManager.prototype;
+
+Class.getUser = function(userId) {
+  return this.users[userId];
+};
+
+Class.createUser = function(userId) {
+  return this.users[userId] = [];
+};
+
+/**
+ * Add a client to the chat room system
+ * @param {Client} client
+ */
+Class.addClient = function addClient(client) {
```

Comment:
It's more for `jsdocs` essentially since it doesn't document the `Class.addClient` nicely. Also the stack trace will have the function name so it's clearer. The additional behaviour is that the function `addClient` can be called within the module without an `RoomManager` object.